### DISCUSSION TITLE
支持更改节点开始时间

### DISCUSSION THREAD

**Discussion by Yersmagit** [Class-Widgets/Class-Widgets#303] [Feature]
### 检查清单

- [X] 我已更新到最新版 ![最新的版本号](https://img.shields.io/github/v/release/RinLit-233-shiroko/Class-Widgets?include_releases&style=flat-square&label=) ，并看过[最新提交](https://github.com/RinLit-233-shiroko/Class-Widgets/commits/main/)，确认我想要的功能还没有实现。
- [X] 我已在 [Issues](https://github.com/RinLit-233-shiroko/Class-Widgets/issues?q=label%3ABug) 中检索，确认这一 Bug 未被提交过。
- [X] 我认为我想要的功能是用户广泛需要的，插件不能替代的，且没有超出 **软件本来** 的开发目标，而非 **添加与课表及教学辅助无关的内容** 。
- [X] 我已知晓由于开发者时间精力有限，我提交的功能请求可能需要比较久的时间才能实现。

### 背景与动机

在最新的库（未打包发布的 v1.1.7 - b4, commit: fdf1e05fbdab2e5b8af53e1ed60c3d24c9b50bdb）中，我发现当用户尝试在时间线编辑界面中删除一个节点时，会弹出提示，确认后会将这个节点和此节点对应的课程与课间一并删除。
这个功能可以帮助减少课表文件的冗余部分，但也随之产生了一个新的问题：如果用户只想修改节点的开始时间，那他应该怎么做呢？
在以前，由于没有节点编辑功能，用户只能通过 先删除原有的节点，再添加一个新的、与原先节点名称相同的节点 的方式来修改；但现在，删除原有的节点会直接导致此节点对应的课程与课间一并被删除，修改节点的开始时间变得更加不便。

### 想要实现或优化的功能

提供一个对已添加的节点的编辑按钮

### 最后一步

- [X] 我认为上述的描述已经足以详细，以便开发人员理解您想要的功能。如果我的 issue 没有按照上述的要求填写，可能会被无条件关闭。

**Comment by RinLit-233-shiroko** [comment]
好的
